website/frontend/templates/docs/tags.html

```diff
@@ -165,41 +177,126 @@ <h1 id="basic_tags">Basic Tags</h1>
-            title="MusicBrainz Identifier">MusicBrainz Identifier</a></td>
+              title="MusicBrainz Identifier">MusicBrainz Identifier</a></td>
         </tr>
         <tr>
           <td>musicbrainz_albumid</td>
           <td> <a href="http://musicbrainz.org/doc/Release" title="Release">Release</a>  <a href="http://musicbrainz.org/doc/MusicBrainz_Identifier"
-            title="MusicBrainz Identifier">MusicBrainz Identifier</a></td>
+              title="MusicBrainz Identifier">MusicBrainz Identifier</a></td>
         </tr>
         <tr>
           <td>musicbrainz_albumartistid</td>
           <td> <a href="http://musicbrainz.org/doc/Release_Artist" title="Release Artist">Release Artist</a>'s
-            <a href="http://musicbrainz.org/doc/MusicBrainz_Identifier" title="MusicBrainz Identifier">MusicBrainz Identifier</a></td>
+              <a href="http://musicbrainz.org/doc/MusicBrainz_Identifier" title="MusicBrainz Identifier">MusicBrainz Identifier</a></td>
         </tr>
         <tr>
           <td>musicbrainz_discid</td>
           <td> <a href="http://musicbrainz.org/doc/Disc_ID" title="Disc ID">Disc ID</a> if album added using CD
-            Lookup (since Picard 0.12)</td>
+              Lookup (since Picard 0.12)</td>
+        </tr>
+        <tr>
+          <td>musicbrainz_originalartistid</td>
+          <td>Original Track Artist's MusicBrainz Identifier</td>
+        </tr>
+        <tr>
+          <td>musicbrainz_originalalbumid</td>
+          <td>Original Release's MusicBrainz Identifier</td>
+        </tr>
+        <tr>
+          <td>musicbrainz_recordingid</td>
+          <td></td>
+        </tr>
+        <tr>
+          <td>musicbrainz_workid</td>
+          <td>Work Name MusicBrainz Identifier</td>
+        </tr>
+        <tr>
+          <td>musicbrainz_releasegroupid</td>
+          <td>Release Group's MusicBrainz Identifier</td>
+        </tr>
+        <tr>
+          <td>titlesort</td>
+          <td><a href="http://musicbrainz.org/doc/Track_Title" title="Track Title">Track Title</a>'s Sort Name</td>
+        </tr>
+        <tr>
+          <td>albumsort</td>
+          <td>Release Title's Sort Name</td>
+        </tr>
+        <tr>
+          <td>musicip_puid</td>
+          <td>MusicIP PUID’s associated with the track. </td>
+        </tr>
+        <tr>
+          <td>musicip_fingerprint</td>
+          <td>MusicIP's Fingerprint</td>
+        </tr>
+        <tr>
+          <td>acoustid_id</td>
+          <td><a href="https://musicbrainz.org/doc/AcoustID">Acoustid</a> associated with the track.</td>
+        </tr>
+        <tr>
+          <td>acoustid_fingerprint</td>
+          <td><a href="https://musicbrainz.org/doc/AcoustID">Acoustid</a> Fingerprint</td>
+        </tr>
+        <tr>
+          <td>bpm</td>
+          <td>Beats per minute</td>
+        </tr>
+        <tr>
+          <td>key</td>
+          <td>Contains the musical key in which the sound starts</td>
```

Comment:
"in which the sound starts"? This seems incorrect. It should either be the key of the entire Track/Recording, either as an "average" or a list (depending on how the tag works).

(A musical piece will start with a note (or multiple notes), not a key. While the key of a piece of music is often the same as the first chord of it, you can't assume that.)